### PR TITLE
Review 1

### PR DESCRIPTION
**Pull request by joein**
pull request for #1

### CODE REVIEW

**Review by akhtyamovpavel — APPROVED** [review]

File: Dockerfile [hunk 1]
```diff
@@ -3,9 +3,9 @@ FROM python:3.8-alpine as base
 FROM base as builder
 COPY requirements.txt ./
 
-RUN apk update; apk add build-base;\
-pip3 install virtualenv==20.0.33;\
-virtualenv -p /usr/local/bin/python3.8 /venv --always-copy;\
+RUN apk update && apk add build-base && \
```
Comment (joein) [inline Dockerfile:6]: rgdfh

File: bot.py [hunk 2]
```diff
@@ -9,32 +9,37 @@
...
 from fetcher import Fetcher
 from constants import FETCHER
-from base_handlers import MiscHandler
-from conv_handler_builder import ConversationHandlerBuilder
+from handlers.entity_handlers import MiscHandler
+from handlers.conversation_handlers import (
+    CharactersConversationHandler,
+    ComicsConversationHandler,
+    EventsConversationHandler,
+    SeriesConversationHandler,
```
Comment (joein) [inline bot.py:17]: dewa

Comment (joein) [inline bot.py:17]: so start

File: bot.py [hunk 3]
```diff
@@ -9,32 +9,37 @@
...
+    dispatcher.bot_data[FETCHER] = fetcher
 
-    characters_conv = ConversationHandlerBuilder.characters()
-    comics_conv = ConversationHandlerBuilder.comics()
-    events_conv = ConversationHandlerBuilder.events()
-    series_conv = ConversationHandlerBuilder.series()
+    characters_handler = CharactersConversationHandler.get()
+    comics_handler = ComicsConversationHandler.get()
+    events_handler = EventsConversationHandler.get()
+    series_handler = SeriesConversationHandler.get()
```
Comment (joein) [inline bot.py:29]: create

**Review by joein — COMMENTED** [review]
finisj

**Pull request merged**